### DISCUSSION TITLE
《开源之迷》参考材料汇总 - 开源之道

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#6] [Announcements]
# 《开源之迷》参考材料汇总 - 开源之道

https://opensourceway.community/posts/book-of-open-source/the-open-source-trilogy/the-fascinating-of-open-source-references/

**Comment by ZoomQuiet** [comment]
#是也乎,
其实, 这只是简单罗列, 读者可能更加期待的:

- 所有链接汇集起来, 变成 开源之迷.别册, 电子书, 免费公开出来
- 所有链接内容关键词, 形成的关联网络, 可能是什么样? 如何理解? 有什么内在关联发现?
- 所有链接作者的社会关联? 研究领域/职能关系?
- 这么多资料链接都是如何收集到的, 用什么管理的? 编撰入图书的版本变化? 也就是说, 如果曾经收集到过 1000个链接, 开始想放到书中 420个, 最终只有242个, 那么中间版本变化如何? 为什么?
- ...

是的, 开源的成品当然重要, 更加重要的是过程以及其中包含的经验...

**Reply by lijiangsheng1** [reply]
这个得从《开源之迷》的整体框架入手啊～ 否则说不通。
